config/default.yml

```diff
@@ -1,5 +1,6 @@
 AllCops:
   NewCops: disable
+  TargetRubyVersion: 2.7
```

Comment:
Not setting this now fails the tests because the implicit Ruby version for the project (`2.7.2` set in `.ruby-version`) doesn't match the lower version in the gemspec (`>= 2.6`).

I _think_ the right thing to do here would be to downgrade the Ruby version of the gem to the lowest possible Ruby of downstream depending apps, but not sure whether that might cause other issues 😕

Reply:
Oh dear that's an annoying gotcha. 

I think your idea of setting the .ruby-version to 2.6 is the right one to match the oldest version of Ruby this project supports. I don't think that'll cause any issues for this project and only imagine it could cause some internal process ones for GDS around upgrading Ruby versions. It does seem Ruby best practice to have the .ruby-version of a gemfile as the lowest supported version.